### ISSUE TITLE
Delete rows in ngx datable removes random controls 

### ISSUE THREAD

**Issue by SudeshAK** [ngx-formly/ngx-formly#2045]
Hi,
I have created a datatable with help of ngx datatable integration example,I have added a 'remove'  button in each row of datatable now when I try to delete some middle row of datatable, random controls inside the rows disappear.
example
https://stackblitz.com/edit/angular-gm1rss

**Comment by aitboudad** [comment]
Please make sure you're using our latest version https://stackblitz.com/edit/angular-gm1rss-kcztzu

**Comment by SudeshAK** [comment]
@aitboudad can you please suggest any alternative fix for this? I am middle of the so upgrading to latest version at this moment will create problem.

**Comment by aitboudad** [comment]
Sorry 🙏, but I can't go back to the resolved issues (lack of time) the only suggestion I would say is to update `formly` to the latest version `5.5.10`.

> at this moment will create problem

which problem ?

**Comment by SudeshAK** [comment]
The problem is I am in middle of the release I will not be able to upgrade any plugin. Upgrade might cause impact and it need to go full round of testing cycle.

**Comment by FritzHerbers** [comment]
@aitboudad 
Some time ago I had the same side-effect that `formly-fields` where not shown when using the following code in the datatable type.

```
  prePopulate(field: FormlyFieldConfig) {
    if (field.fieldArray && field.fieldArray.fieldGroup) {
      field.fieldArray.fieldGroup.forEach(element => {
        element.wrappers = [];    // changing to undefined, all is OK
        element.templateOptions.hideLabel = true;
      });
    }
  }
```

The first intention was to hide the label in `formly-wrapper-form-field`. Later decided in table view-only mode, that the rows had too much margin because of `form-field.wrapper` and added `wrappers` to `[]`.

When later adding code for removing rows, I noticed that some `formly-fields` where not shown when deleting a row, similar as the OP. When changing `wrappers `from `[]` to `undefined`, all fields where shown correct.

I took your SB and modified it with the code above to show the error on latest formly version:
https://stackblitz.com/edit/angular-gm1rss-fgfnoj?file=src/app/datatable.type.ts
To reproduce, delete one of the first rows.

You might add some defensive checks to allow `wrapper = []` in prePopulate.
